bugrap/src/main/java/org/vaadin/bugrap/ui/columns/CamelcaseTextRenderer.java

```diff
@@ -0,0 +1,32 @@
+package org.vaadin.bugrap.ui.columns;
+
+import com.vaadin.ui.renderers.TextRenderer;
+
+import elemental.json.JsonValue;
+
+public class CamelcaseTextRenderer extends TextRenderer {
+
+	@Override
+	public JsonValue encode(Object value) {
+		if (value == null)
+			value = getNullRepresentation();
+		return super.encode(converToCamelCaseString(value.toString()));
+	}
+
+	private String converToCamelCaseString(String str) {
+		String[] words = str.split(" ");
+		StringBuffer buffer = new StringBuffer();
```

Comment:
In this case, Instead of using _StringBuffer_ it's better to use _StringBuilder_.